openerp/http.py

```diff
@@ -1326,7 +1328,7 @@ def save_request_data(self):
         #      servers) can access them. It also allow stale files to be deleted by `session_gc`.
         for f in req.files.values():
             storename = 'werkzeug_%s_%s.file' % (self.sid, uuid.uuid4().hex)
-            path = os.path.join(root.session_store.path, storename)
+            path = os.path.join(openerp.tools.config.session_dir, storename)
```

Comment:
I think using `root.session_store.path` was correct here. We just need a custom base class for SessionStore that declares self.path = tempfile.gettempdir() if odoo.tools.config.session_dir wasn't set, like how FilesystemSessionStore did it.